Change: value8 => valueX

@package vaimo/composer-patches-target1
@assert vaimo/composer-patches-target1,value8,valueX

--- src/example.txt.org
+++ src/example.txt
@@ -5,7 +5,7 @@
 value5
 value6
 value7
-value8
+valueX
 value9
 value10
 value11
